### DISCUSSION TITLE
__webpack_hmr error 404

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#4588] [Help / Questions] [answered]
Hi,

I'm running the WikiJS on the server.
Making the service as the installation manual tells to.
But I got the error 404 on the url/__webpack_hmr.

Any suggestion ?

Thank you in advance.
Thitinan

**Comment by NGPixel** [comment]
How did you install Wiki.js exactly? webpack_hmr is only for development and would not occur when using the published versions.

**Comment by ghost** [comment, marked as answer]
Hi,

Thank you for your reply.

Turned out the problem is I ran the yarn dev while changing the source on the dev machine, then export the data/assets to the production, and it is the dev compiled version, so __webpack_hmr followed the build.
I did yarn build, and deploy again. The error was gone, but I got the message this is the development... not production something.

So as the title goes, the thread is solved.
Sorry for inconvenient.
